stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/experimental/plugin/factory/StreamOfflinePluginFactory.kt

```diff
@@ -88,6 +91,19 @@ public class StreamOfflinePluginFactory(
         val userStateFlow = MutableStateFlow(ChatClient.instance().getCurrentUser())
         val stateRegistry = StateRegistry.getOrCreate(scope, userStateFlow, repos, repos.observeLatestUsers())
         val logic = LogicRegistry.getOrCreate(stateRegistry)
+        val globalState = GlobalMutableState.getOrCreate()
```

Comment:
This variable is already present in this method, it is called `globalStateRegistry`. The name doesn't make much sense... feel free to fix it.